CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-Thank you for contributing to the docs! We have a few guidelines which will allow your PR to pass our CI checks and successfully merge.
+Thank you for contributing to the docs! We have a few guidelines which will allow your patch: pull request (PR) to pass our CI checks and successfully merge.
```

Comment:
@laurentava  patch: pull request?
PR is normally called pull request. What do you mean by `patch: pull request (PR)`

Reply:
@vipulgupta2048 my edit was to change PR to the full word Pull Request (as I didn't know what it meant at first). I figured writing it down in full words the first time is helpful for people like me who are not very technical. I thought per the instructions when you make a edit you putt  patch:  before the edit? If not, then I would love help on how to make the edit correctly. Thanks!!

Reply:
@laurentava Yes, you are absolutely correct in that. I agree we should have a full form of pull request in the document early on for folks to know about it. What I meant was, 

The full form of PR is just `Pull Request`. The part about adding `patch: ` is described later and isn't needed here is what I mean.

Reply:
@vipulgupta2048 ok I understand. I fixed it. Let me know if it's ok! Thanks!